Readme.txt

```diff
@@ -1,77 +1,138 @@
                                 MORSE RUNNER
                               Contest Simulator
                                   freeware
-
-                   Version 1.80 Field Day Contest Prototype
+                                  
+                        Version 1.82 - ARRL DX Contest
+            The third release of the Morse Runner Community Edition
```

Comment:
Not sure this line adds value long term.

Reply:
long term -- like in the future readme where it changes to "The fourteenth release of the Morse Runner Community Edition"?  

I'm viewing the readme as a help document as well as a sales pitch for MRCE awesomeness. :) I thought it was a neat tidbit to be proud of.. Our first year and three versions. We did great!!! Field Day alone was killer but now ARRL DX!!! My job is to brag on you guys. I don't have to be humble. :)

Reply:
okay. These are release notes, not marketing or advertising materials. I good with this.